scripts/common/preflights.sh

```diff
@@ -391,12 +391,19 @@ function preflights_require_no_kubernetes_or_current_node() {
     return 0
 }
 
+HOST_PREFLIGHTS_RESULTS_OUTPUT_DIR="host-preflights"
 function host_preflights() {
     local is_primary="$1"
     local is_join="$2"
     local is_upgrade="$3"
 
     local opts=
+
+    local out_file=
+    out_file="${HOST_PREFLIGHTS_RESULTS_OUTPUT_DIR}/results-$(date +%s).txt"
```

Comment:
won't this be relative to the path that this is being run in? Should we put this with the rest of the kurl cache?

Reply:
fixed